ROS/controls/msg/MotorOrientation.msg

```diff
@@ -0,0 +1,2 @@
+int32 numMotors
+MotorState[] states
```

Comment:
Having one message type that seems to be used for both desired orientation and current orientation may lead to confusion; when naming the topic that uses MotorOrientation.mg make sure to make it very clear if it's desired or current motor orientation. 

If you split this into two different messages for current state and desired, you could include more motor data such as temperature and current draw for the current state message.